ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly-4.16-upgrade-from-stable-4.16.yaml

```diff
@@ -160,13 +316,73 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-nutanix-ipi-proxy
+- as: nutanix-ipi-proxy-fips-minimal-permission-f28
+  cluster: build01
+  cron: 6 15 5 * *
+  steps:
+    cluster_profile: nutanix-qe-dis
+    env:
+      EXTRACT_MANIFEST_INCLUDED: "true"
+      FIPS_ENABLED: "true"
+    test:
+    - chain: openshift-upgrade-qe-test
+    workflow: cucushift-installer-rehearse-nutanix-ipi-proxy-minimal-permission
+- as: nutanix-ipi-zones-f28
+  cluster: build01
+  cron: 16 22 14 * *
+  steps:
+    cluster_profile: nutanix-qe-zone
+    env:
+      EXTRACT_MANIFEST_INCLUDED: "true"
+    test:
+    - chain: openshift-upgrade-qe-test
+    workflow: cucushift-installer-rehearse-nutanix-ipi-zones
+- as: vsphere-8-ipi-ovn-etcd-encryption-f28
```

Comment:
The main test point for the install workflow is to cover vsphere-8, now our vsphere platform is already upgraded to version 8, so suggest to use `vsphere-ipi-compact-etcd-encryption-f28` to replace it.

Reply:
I saw `vsphere-8-ipi-ovn-etcd-encryption-f28` was removed, but no new job - `vsphere-ipi-compact-etcd-encryption-f28` was added, do I miss something?

Reply:
Yes, missed it in above commit and I put them in multi files in last commit